public/index.html

```diff
@@ -2,26 +2,15 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="/public/favicon.ico" />
```

Comment:
J'ai pas l'impression que le favicon fonctionne de mon côté ? (logo par défaut au lieu de celui d'Avastar)

cc. photo ci-dessous 👇

![Screenshot 2022-05-16 at 23 43 38](https://user-images.githubusercontent.com/45219966/168687456-5caf717d-b375-49cd-8271-178f8c5d8bea.png)

Reply:
oui ce n'est pas le logo avastar, il en faut un en format favicon.

Reply:
Ok c'est faisable d'avoir ça non ? P.e. voir avec Julie ou checker si des converteurs en ligne ? 
Pas urgent mais bon nice to have quand même.

Reply:
oui ajouté au "refactoring"